doc/Data Plane Code Generation/DeveloperDrivenEvolution.md

```diff
@@ -0,0 +1,144 @@
+# Developer driven evolution
+To grow up a protocol method that returns raw JSON to a model, steps are:
```

Comment:
Instead of "growing up a protocol method," let's give this its formal name, "adding a convenience API."